### DISCUSSION TITLE
「質問」のフォーマット（「Q&A」と「意見交換」）でテンプレートが変わらない

### DISCUSSION THREAD

**Discussion by lamrongol** [increments/qiita-discussions#373] [要望・アイデア・不具合]
「質問」を投稿する場合テンプレートが表示され、「Q&A」と「意見交換」のフォーマット選択欄がありますがこれを変更してもテンプレートが同じです。

「Q＆A」では「〇〇というエラーが発生するのでどうすれば解決できるか」といった質問がなされ、「意見交換」では例えば「Twitterに財務やヘイトスピーチなどの問題が取り沙汰されているため代替サービスを作りたいがどのWebフレームワークが良いか」のような確定した「正解」が存在しない質問が想定されてると思うのですが、後者では「発生している問題・エラー」や「該当するソースコード」といった項目は必要ないため無くすか別の適切なテンプレートを用意しなければいけないと思いますが、現在フォーマットを変更してもテンプレートが切り替わることはありません。

これはバグでしょうか。それともこちらの想定が間違ってるのでしょうか。
[Qiita ヘルプ](https://help.qiita.com/ja/articles/qiita-question)でも違いについて仕様しか書かれておらず、質問内容によって「Q&A」と「意見交換」のどちらを選ぶべきかが書かれていません。

**Comment by tanaka-asami415** [comment]
@lamrongol 
ご意見ありがとうございます🙏

現在、テンプレートは変わらない仕様になっておりますが、切り替えについて検討させていただきます。

ヘルプに関しては、ご不便をおかけして申し訳ありません。
「意見交換」は、[こちら](https://blog.qiita.com/question-beta-release/)のような内容を想定しておりますが
ヘルプに反映ができていなかったため更新しておきます！🙇
